app/src/main/java/de/westnordost/streetcomplete/quests/police_type/AddPoliceType.kt

```diff
@@ -0,0 +1,31 @@
+package de.westnordost.streetcomplete.quests.police_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.osmquest.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.quest.NoCountriesExcept
+import de.westnordost.streetcomplete.quests.police_type.AddPostboxRoyalCypherForm
+import de.westnordost.streetcomplete.quests.police_type.PostboxRoyalCypher
+
+class AddPoliceType : OsmFilterQuestType<PostboxRoyalCypher>() {
```

Comment:
```suggestion
class AddPoliceType : OsmFilterQuestType<PoliceType>() {
```

Reply:
I forgot to add the file for staging on Git 😂 I think to have solved this one and some of the ones below 🙂